classes/class-pmpro-discount-code-list-table.php

```diff
@@ -0,0 +1,516 @@
+					$item['code']
+				)
+			);
+
+			$delete_nonce_url = wp_nonce_url(
+				add_query_arg(
+					[
+						'page'   => 'pmpro-discountcodes',
+						'delete' => $item['id'],
+					],
+					admin_url( 'admin.php' )
+				),
+				'delete',
+				'pmpro_discountcodes_nonce'
+			);
+
+			$actions = [
+				'id'	 => sprintf(
+					// translators: %s is the Order ID.
+					__( 'ID: %s', 'paid-memberships-pro' ),
+					esc_attr( $item['id'] )
+				),
+				'edit'   => sprintf(
+					'<a title="%1$s" href="%2$s">%3$s</a>',
+					esc_attr__( 'Edit', 'paid-memberships-pro' ),
+					esc_url(
+						add_query_arg(
+							[
+								'page' => 'pmpro-discountcodes',
+								'edit' => $item['id'],
+							],
+							admin_url( 'admin.php' )
+						)
+					),
+					esc_html__( 'Edit', 'paid-memberships-pro' )
+				),
+				'copy'   => sprintf(
+					'<a title="%1$s" href="%2$s">%3$s</a>',
+					esc_attr__( 'Copy', 'paid-memberships-pro' ),
+					esc_url(
+						add_query_arg(
+							[
+								'page' => 'pmpro-discountcodes',
+								'edit' => - 1,
+								'copy' => $item['id'],
+							],
+							admin_url( 'admin.php' )
+						)
+					),
+					esc_html__( 'Copy', 'paid-memberships-pro' )
+				),
+				'delete' => sprintf(
+					'<a title="%1$s" href="%2$s">%3$s</a>',
+					esc_attr__( 'Delete', 'paid-memberships-pro' ),
+					'javascript:pmpro_askfirst(\'' . esc_js( $delete_text ) . '\', \'' . esc_js( $delete_nonce_url ) . '\'); void(0);',
+					esc_html__( 'Delete', 'paid-memberships-pro' )
+				),
+			];
+
+			if ( 0 < (int) $item['uses'] ) {
+				$actions['orders'] = sprintf(
+					'<a title="%1$s" href="%2$s">%3$s</a>',
+					esc_attr__( 'View Orders', 'paid-memberships-pro' ),
+					esc_url(
+						add_query_arg(
+							[
+								'page'          => 'pmpro-orders',
+								'discount-code' => $item['id'],
+								'filter'        => 'with-discount-code',
+							],
+							admin_url( 'admin.php' )
+						)
+					),
+					esc_html__( 'Orders', 'paid-memberships-pro' )
+				);
+			}
+
+			/**
+			 * Filter the extra actions for this discount code.
+			 *
+			 * @since TBD
+			 *
+			 * @param array  $actions The list of actions.
+			 * @param object $code    The discount code data.
+			 */
+			$actions = apply_filters( 'pmpro_discountcodes_row_actions', $actions, $item );
+
+			$actions_html = [];
+
+			foreach ( $actions as $action => $link ) {
+				$actions_html[] = sprintf(
+					'<span class="%1$s">%2$s</span>',
+					esc_attr( $action ),
+					$link
+				);
+			}
+
+			if ( ! empty( $actions_html ) ) {
+				echo implode( ' | ', $actions_html );
+			}
+		?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Render the discount uses value
+	 *
+	 * @param array  $item
+	 *
+	 * @return mixed
+	 */
+	public function column_uses( $item ) {
+		
+		global $wpdb;
+		
+		
+
+		if( $item['uses'] > 0 ) {
+			echo "<strong>" . (int)$item['used'] . "</strong>/" . $item->uses;
```

Comment:
```suggestion
			echo "<strong>" . (int)$item['used'] . "</strong>/" . (int)$item['uses'];
```
Fixing PHP warning for uses > 0